### ISSUE TITLE
Version 110 is not supported on OS X

### ISSUE THREAD

**Issue by killkrt** [tomaka/glium#754]
Hi,

I'm getting this issue with almost examples (excluding "manual-creation"), they seems to create a window and then crash with the following error:

```
thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:2: \'\' :  version \'110\' is not supported\nERROR: 0:3: \'\' :  #version required and missing.\nERROR: 0:4: \'attribute\' : syntax error: syntax error\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:774
```

These are my system info:
OS: OS X 10.10.3
clang: Apple LLVM version 6.1.0 (clang-602.0.49) (based on LLVM 3.6.0svn)
rustc: 1.0.0-beta (9854143cb 2015-04-02) (built 2015-04-02)

**Comment by tioover** [comment]
+1 a long time 
excluding blit

**Comment by tomaka** [comment]
Could you please check that it works now?

**Comment by killkrt** [comment]
Hi Toamaka,

Thank for your fixing, unfortunately I still got errors.
These are the errors when I build test with `cargo test`:

```running 51 tests
_RegisterApplication(), FAILED TO REGISTER PROCESS WITH CPS/CoreGraphics in WindowServer, err=-50
2015-04-19 18:18:22.441 attrs-97a636aeeb219f6d[8669:840807] setGestureEventMask: InstallApplicationEventHandler failed with error -9866
_RegisterApplication(), FAILED TO REGISTER PROCESS WITH CPS/CoreGraphics in WindowServer, err=-50
2015-04-19 18:18:22.443 attrs-97a636aeeb219f6d[8669:840802] setGestureEventMask: InstallApplicationEventHandler failed with error -9866
_RegisterApplication(), FAILED TO REGISTER PROCESS WITH CPS/CoreGraphics in WindowServer, err=-50
attrs-97a636aeeb219f6d(8669,0x113c04000) malloc: *** error for object 0x7f8e3af05fb0: double free
*** set a breakpoint in malloc_error_break to debug
Process didn't exit successfully: `/Users/KillKRT/Documents/Development/rust/glium/target/debug/attrs-97a636aeeb219f6d` (signal: 6)```

And these are the errors for all the examples:

Test name | Error
--------------|----------
deferred | ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:18: Invalid call of undeclared identifier \'texture2D\'\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```
displacement_mapping | ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:2: \'\' :  version \'430\' is not supported\nERROR: 0:3: \'\' :  #version required and missing.\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```
fullscreen | ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:2: \'\' :  version \'110\' is not supported\nERROR: 0:3: \'\' :  #version required and missing.\nERROR: 0:6: \'attribute\' : syntax error: syntax error\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```
image | ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:8: Invalid call of undeclared identifier \'texture2D\'\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```
instancing | ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:2: \'\' :  version \'110\' is not supported\nERROR: 0:3: \'\' :  #version required and missing.\nERROR: 0:4: \'attribute\' : syntax error: syntax error\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```
screenshot | ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:4: \'output\' : Reserved word. \nERROR: 0:4: \'output\' : syntax error: syntax error\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```
tessellation | ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:7: Use of undeclared identifier \'gl_FragColor\'\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```
triangle | ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:4: \'output\' : Reserved word. \nERROR: 0:4: \'output\' : syntax error: syntax error\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```
tutorial-01 | ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:5: Use of undeclared identifier \'gl_FragColor\'\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```

While `blitting`and `teapot`are working correctly

System info:
OS: OS X 10.10.3
rustc: rustc 1.0.0-beta.2 (e9080ec39 2015-04-16) (built 2015-04-16)

**Comment by tomaka** [comment]
:cry: Thanks for the detailed report, I didn't think it was so bad.

**Comment by tomaka** [comment]
Could you try again?

**Comment by killkrt** [comment]
Good! Now they work! :ok_hand:
Thank a lot!

There is only one that is not working: `displacement_mapping` that fails with:
```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: CompilationError("ERROR: 0:15: Use of undeclared identifier \'gl_FragColor\'\n")', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/result.rs:750```

And `fullscreen` fails when I push `[Enter]`with the following error:
```thread '<main>' panicked at 'not yet implemented', /Users/KillKRT/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.0.25/src/cocoa/mod.rs:316```

**Comment by tomaka** [comment]
The displacement mapping example has been fixed in #812
